packages/dev/core/src/Gizmos/gizmo.ts

```diff
@@ -176,6 +191,7 @@ export class Gizmo implements IGizmo {
 
     protected _updateGizmoRotationToMatchAttachedMesh = true;
     protected _updateGizmoPositionToMatchAttachedMesh = true;
+    protected _targetAnchorPoint = GizmoAnchorPoint.Origin;

```

Comment:
I'd use gizmo or simply `anchorPoint` instead of `targetAnchorPoint`

Reply:
Sure, agree with that and updated the name